base/steer/FairRun.h

```diff
@@ -199,6 +199,7 @@ class FairRun : public TNamed
     virtual void SetSource(FairSource* tempSource);
     /** Return non-owning pointer to source **/
     FairSource* GetSource() { return fSource.get(); }
+    FairRootManager* GetRootManager() const { return fRootManager; }
```

Comment:
```suggestion
    FairRootManager& GetRootManager() const { return fRootManager; }
```

I guess, we can make sure to always return one, right?

Reply:
Yes, we assume that anyway everywhere. And we're quiet sure also.

That said, most of the calling code expects a pointer. That said, let's see, whether we can rewrite them to use references!

Reply:
done.